robolectric/src/test/java/org/robolectric/plugins/SQLiteModeConfigurerTest.java

```diff
@@ -43,7 +43,7 @@ public void osArchSpecificConfig() {
       SQLiteModeConfigurer configurer2 =
           new SQLiteModeConfigurer(System.getProperties(), new PackagePropertiesLoader());
 
-      assertThat(configurer2.defaultConfig()).isSameInstanceAs(Mode.LEGACY);
+      assertThat(configurer2.defaultConfig()).isSameInstanceAs(Mode.NATIVE);
```

Comment:
Maybe this whole test can be removed now, since there's no specific OS-specific logic anymore?

Reply:
I've removed it